requirements-dev.txt

```diff
@@ -1,19 +1,19 @@
 -r requirements.txt
 
 # For tests
-git+https://github.com/alphagov/digitalmarketplace-test-utils.git@1.4.0#egg=digitalmarketplace-test-utils==1.4.0
-coverage==3.7.1
-flake8-per-file-ignores==0.4.0
+coverage
+coveralls
 flake8==3.5.0
-freezegun==0.3.8
-hypothesis==2.0.0
-mock==2.0.0
-pycodestyle==2.3.0
-pytest-cov==2.5.1
-pytest==3.2.3
-python-coveralls==2.5.0
-requests-mock==0.6.0
-testfixtures==6.0.2
+flake8-per-file-ignores
+freezegun
+hypothesis<4.0.0,>=3.0.0
+mock
+pytest
+pytest-cov
+requests-mock
+testfixtures<7.0.0,>=6.0.2
+
```

Comment:
I'm getting less and less keen on the removal of version pinning. And I was never _keen_.

Reply:
Discuss at stand-up?

Reply:
Okay, agreement is that we will pin dev requirements, but @jonodrew will look at setting up something like depend-a-bot to keep our dependencies up-to-date.